test/integration/proxy-config.test.js

```diff
@@ -1,13 +1,18 @@
 var expect = require('expect.js'),
-    ProxyConfig = require('../../lib/index.js').ProxyConfig;
+    ProxyConfig = require('../../lib/index.js').ProxyConfig,
+    DEFAULT_PATTERN = 'http+https://*/*',
+    DEFAULT_HOST = '',
+    DEFAULT_PORT = 8080;
 
 /* global describe, it */
 describe('Proxy Config', function () {
     it('should initialize the values to their defaults', function () {
         var p = new ProxyConfig();
 
-        expect(p.match.pattern).to.be('<all_urls>');
```

Comment:
@SamvelRaja The default of `<all_urls>` for ProxyConfig has now been replaced with a more sane default which accepts all protocols, all hosts, and all paths.